server/builder/receiver.js

```diff
@@ -1,19 +1,17 @@
-const Queue = require('./queue')
+const Queue = require('../rabbitmq')
```

Comment:
Hmm I'm not sure to understand why this file is here. It calls `./nbuild.py` but is in the `server/` directory. Am I missing something?

Reply:
builder folder deleted. It was added by the rebase.